src/AutoRest.CSharp/Mgmt/Generation/MgmtExtensionWriter.cs

```diff
@@ -59,7 +59,7 @@ public MgmtExtensionWriter(CodeWriter writer, MgmtExtensions extensions, BuildCo
 
         protected Parameter ExtensionParameter { get; }
 
-        protected void WriteMethodWrapper(MgmtClientOperation clientOperation, bool async)
+        protected void WriteMethodWrapper(MgmtClientOperation clientOperation, bool isAsync)
```

Comment:
Nit/Curious: why this change?
We have a convention elsewhere to use async, wbn to be consistent: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Pipeline/BearerTokenAuthenticationPolicy.cs#L113

Reply:
bool should be verbs that ask a question IMHO, I do see a mixed style throughout the code